source/scim/azure.rst

```diff
@@ -1,43 +0,0 @@
-:orphan:
-
-Azure portal
-============
-
-References:
------------
-
--  `Use scim to provision users and groups <https://learn.microsoft.com/fr-fr/azure/active-directory/app-provisioning/use-scim-to-provision-users-and-groups#integrate-your-scim-endpoint-with-the-azure-ad-provisioning-service>`_
-
-Setup
------
-
--  Connect to your `Azure portal <https://aad.portal.azure.com/>`_ and  go to the **Azure Active Directory** > **Enterprise applications** menu.
--  Select **New application** and choose **Create your own application**.
--  Choose a name for your application, select **Integrate any other application you don’t find in the gallery (Non-gallery)** and click on the **[Create]** button.
-
-.. figure:: pics/azure-1.png
-   :alt: Azure create app
-
--  In the “Provisioning” section,
-
-   -  Select **Automatic**,
-   -  Fill **Tenant Url** with the API url of your SCIM server you copied from the :ref:`plugin configuration <setup_scim>`.
-   -  Optionally, fill **Secret** field:
-      For Azure, the awaited secret is a long life valid jwt token.
-      We cannot use an oauth exchange (Azure doesn’t ask for an authorize URL).
-      So in GLPI, setup you SCIM server with **Bearer** security and paste the JWT token from GLPI in the **Secret token** field of Azure.
```

Comment:
I think we lose this important information when migrating to "Entra" document.
As we can't use a real OAuth exchange with this provider, we must explicitly say it.
The current "Make sure you paste the token (Jwt token) to ensure your application works properly." is not sufficient.

Reply:
No, I want the full sentence like before. The last is too simple and does not explain why we need a long-life JWT token (it's a bad practice to have this kind of token as they can be stolen)